### DISCUSSION TITLE
Procedural animation support

### DISCUSSION THREAD

**Discussion by Dinesh0N** [friction2d/friction#259] [Ideas]
Procedural amination support like blender geometry node system and Inkscape path effects

**Comment by rodlie** [comment]
> Procedural amination support like blender geometry node system

We have expression support and [shader effects](https://github.com/friction2d/friction-shader-plugins) support (both with support for JS).

basic expr example:

https://github.com/user-attachments/assets/630acde9-1994-41d3-9f40-5161fb26b861

> and Inkscape path effects

We have path effects (and more).

![friction-path-effects](https://github.com/user-attachments/assets/456ef345-89ea-44e8-ba3f-b163554445eb)
